apps/ledger-live-desktop/src/renderer/screens/manager/DeviceDashboard/AppsList/index.tsx

```diff
@@ -125,6 +127,17 @@ const AppsList = ({
 
   const displayedAppList = isDeviceTab ? device : catalog;
 
+  const { getFeature } = useFeatureFlags();
+  const enabledAppList = useMemo(
+    () =>
+      displayedAppList.filter(({ currencyId }: App) => {
```

Comment:
Yes, it can be inferred, but there is a destruction here, adding the type can make the code more readable. Imaging `{ id }: SomeComp`, it will make the code reader confused in the code scope, whose `id` it is.